test/unit/app/utils/test_phone.py

```diff
@@ -3,15 +3,36 @@
 """
 import pytest
 
+from server.utils.phone import ChannelType
 
-@pytest.mark.parametrize("phone,expected", [
-    ("0401391419", "+61401391419"),
-    ("+961401391419", "+961401391419"),
+
+@pytest.fixture(scope='function')
```

Comment:
Nice. It's a lot cleaner having the fixtures in the file they're actually used in (given many are only used in one file)